src/docs/command-line.dox

```diff
@@ -469,8 +469,12 @@ The \c upgrade command has no command-specific options.
 @section util_verify wt verify
 Check the structural integrity of a table.
 
-The \c verify command verifies the specified table, exiting success if
-the data source is correct, and failure if the data source is corrupted.
+The \c verify command verifies the specified table if a URI argument is provided,
+exiting success if the data source is correct, and failure if the data source is corrupted.
+If no URI argument is provided, it verifies all the tables in the database. Verify will
+keep going and try to verify the rest of the tables in the scenario that it encounters an
+error when verifying a table. Currently, the verify operation is not supported with tiered
+storage and the history store file.
```

Comment:
> If no URI ... error when verifying a table

I think this would be simpler as:

> If no URI argument is provided, it verifies each table in the database, reporting on any that are damaged.

Also, please change `tiered storage` to `tiered tables`.